### PR TITLE
feature/asm-3517_labels - support custom labels in helm charts

### PR DESCRIPTION
**Pull request by liorhasson**

### CODE REVIEW

**Review by niv-ez — APPROVED** [review]

**Review by OriBenHur-akeyless — APPROVED** [review]

**Review by seank-akeyless — APPROVED** [review]
LGTM. Thank you.

File: charts/akeyless-api-gateway/values.yaml [hunk 1]
```diff
@@ -20,6 +20,7 @@ httpProxySettings:
 
 deployment:
   annotations: {}
+  labels: {}
```
Comment (devorbitus) [inline charts/akeyless-api-gateway/values.yaml:23]:
```suggestion
  ## Labels should be in the form of key: value
  ## labels:
  ##   my-label-key: my-label-value
  labels: {}
```

File: charts/akeyless-api-gateway/values.yaml [hunk 2]
```diff
@@ -43,6 +44,7 @@ deployment:
 service:
   # Remove the {} and add any needed annotations regarding your LoadBalancer implementation
   annotations: {}
+  labels: {}
```
Comment (devorbitus) [inline charts/akeyless-api-gateway/values.yaml:47]:
```suggestion
  ## Labels should be in the form of key: value
  ## labels:
  ##   my-label-key: my-label-value
  labels: {}
```

File: charts/akeyless-api-gateway/values.yaml [hunk 3]
```diff
@@ -89,6 +91,7 @@ ingress:
   enabled: false
 
   annotations: {}
+  labels: {}
```
Comment (devorbitus) [inline charts/akeyless-api-gateway/values.yaml:94]:
```suggestion
  ## Labels should be in the form of key: value
  ## labels:
  ##   my-label-key: my-label-value
  labels: {}
```

**Review by devorbitus — CHANGES_REQUESTED** [review]
better comment suggestions for the new feature IMO

**Review by OriBenHur-akeyless — APPROVED** [review]

**Pull request merged**